src/lib.rs

```diff
@@ -293,17 +305,28 @@ impl Preprocessor for Bibiography {
             let bibliography = build_bibliography(bib_content?);
             if bibliography.is_err() {
                 warn!(
-                    "Error buildig Bibliography from raw content. Skipping render: {:?}",
+                    "Error building Bibliography from raw content. Skipping render: {:?}",
                     bibliography.err()
                 );
                 return Ok(book);
             }
 
             let bib = bibliography.unwrap();
 
-            Bibiography::expand_cite_references_in_book(&mut book, &bib);
+            let cited = Bibiography::expand_cite_references_in_book(&mut book, &bib);
 
-            let html_content = Bibiography::generate_bibliography_html(&bib);
+            let cited_only = match cfg.get("render-bib") {
```

Comment:
This is only an interim solution. I would suggest to parse the entire config into a struct to avoid `cfg.get(...)` all over the code.

Reply:
yes. I was thinking on that too

Reply:
I would prefer to do it in a separate MR

Reply:
Yes